android-components/components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -666,6 +666,16 @@ class GeckoEngine(
                 field = value
             }
 
+        override var cookieBannerHandlingDetectOnlyMode: Boolean = false
```

Comment:
There is a `CookieBannerHandlingMode.DETECT_ONLY` so why is this a boolean with a runtime setting for detect only mode? It seems like it should be set the same as the other modes.